build.gradle

```diff
@@ -35,6 +34,7 @@ configure(allprojects) { project ->
             mavenCentral()
             jcenter()
             maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
+            maven { url 'https://plugins.gradle.org/m2/'}
```

Comment:
Why do we need this?

Reply:
this url is added for the coverall upgrade